src/EventTeachingService.h

```diff
@@ -7,29 +7,30 @@
 
 #include "Service.h"
 
-namespace VLCB
-{
+namespace VLCB {
```

Comment:
Please keep curly brackets "{" on new lines. This is the style we want to use.
Have you set this style in Notepad++? Or is this in Arduino IDE?

Reply:
This is the Arduino IDE and it's a pain!  If I use its auto format, I have to go through and correct the misplaced opening brace and sometimes I miss them.  I try to get the formatting all done in Notepad++ now.